test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
```

Comment:
The problem I want to solve is the friction in running the tests. I want all of the following to work (with this PR, they do):

* ruby test/basic_test.rb (doesn't work on master)
* rake (doesn't work on master)
* bundle exec rake (works on master)
* bundle exec ruby test/basic_test.rb (doesn't work on master).

If you want to insist on `bundle exec` being used, then I at least suggest:
```ruby
if not defined?(Bundler)
  warn "Please prefix `bundle exec` in order to run this command."
  exit(false)
end
```
instead of a confusing backtrace. I think the day of managing gem dependencies manually, from a set of system gems, is gone and counter-productive for the majority of developers. I think trying to run with and without bundler makes development more complicated, and the arguments for doing so usually aren't that good. So, if you accept Bundler is part of the project to run the tests, then i think at the least the development experience can improved.